source/MaterialXTest/MaterialXRender/RenderUtil.cpp

```diff
@@ -383,12 +384,6 @@ bool ShaderRenderTester::validate(const mx::FilePathVec& testRootPaths, const mx
                                 mx::ImageVec imageVec;
                                 for (size_t f = 0; f < wedgeFiles.size(); f++)
                                 {
-                                    const std::string& wedgeFile = wedgeFiles[f];
-                                    if (wedgeFile != file)
-                                    {
-                                        continue;
-                                    }
-
                                     // Make this a utility
                                     std::string parameterPath = wedgeParameters[f];
```

Comment:
Since `f` index into the wedgeParameters vector here, should the loop iterate on the size of wedgeParameters, and not on wedgeFiles?

Reply:
There is a check to see if they are the same size as the loop iterates over a tuple of { file, parameter, min, max, step } at a time and not 1 file and multiple {parameters,min,max,steps}.

To wedge on another parameter in the same file something like this would be required in the options file:
```
   <parameter name="wedgeFiles" type="string" value="wedge_conductor.mtlx,wedge_conductor.mtlx" />
      <parameter name="wedgeParameters" type="string" value="test_conductor/roughness1/roughness,test_conductor/roughness1/anisotropy" />
      <parameter name="wedgeRangeMin" type="floatarray" value="0.0,0.0" />
      <parameter name="wedgeRangeMax" type="floatarray" value="1.0,1.0" />
      <parameter name="wedgeSteps" type="integerarray" value="4,4" />
```